Installation_Guide/Installation-Guide.rst

```diff
@@ -258,7 +258,7 @@ Note: The URL of the repository must point to the location of the repositories
   echo 'ADD_EXTRA_GROUPS=1' | sudo tee -a /etc/adduser.conf
   echo 'EXTRA_GROUPS=video' | sudo tee -a /etc/adduser.conf
 
-Note: The current release supports CentOS/RHEL v7.6. Before updating to the latest version of the operating system, delete the ROCm packages to avoid DKMS-related issues.
+Note: The current release supports CentOS/RHEL v7.7. Before updating to the latest version of the operating system, delete the ROCm packages to avoid DKMS-related issues.
```

Comment:
I thought we were supporting CentOS/RHEL 8.1 with docker support for CentOS/RHEL 7.7. In addition I thought SLES 15 Service Pack 1 was added.